dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -503,7 +503,7 @@ class ScriptLevelsControllerTest < ActionController::TestCase
 
   test "should 404 for invalid script level for twenty hour" do
     assert_raises(ActiveRecord::RecordNotFound) do # renders a 404 in prod
-      get :show, script_id: Script::TWENTY_HOUR_ID, id: 40000
+      get :show, script_id: Script::TWENTY_HOUR_NAME, id: 40000
```

Comment:
Hm, the way to do that is to use Script.get_from_cache instead of Script.find everywhere. So for that particular case you could update set_script.

I think this particular case might also be going away soon because of Steve's header changes?